packages/ses/src/tame-locale-methods.js

```diff
@@ -39,7 +39,7 @@ const nonLocaleCompare = tamedMethods.localeCompare;
 
 export default function tameLocaleMethods(intrinsics, localeTaming = 'safe') {
```

Comment:
We still need a special case for `Number.toLocaleString`, to avoid the same bug that XS had. Yes?